contracts/token/BiconomyTokenPaymaster.sol

```diff
@@ -44,15 +44,22 @@ contract BiconomyTokenPaymaster is
     using SignatureCheckerLib for address;
 
     // State variables
+
+    // Review: where will the be used since we have pre charge and refund model?
```

Comment:
we can just discuss here on each points by starting a comment like this
@ShivaanshK